backend/social_tasks/sorsa_client.py

```
@@ -107,16 +108,16 @@ def is_following(self, actor_handle: str, target_handle: str) -> tuple[bool, dic
         # Strict type check: a schema change at Sorsa must surface as a loud
         # SorsaError (-> 503, user retries later), not silently read as
         # "not following" for every check.
-        is_following = data.get('is_following')
+        is_following = data.get('follow')
         if not isinstance(is_following, bool):
             raise SorsaError(
-                f'Unexpected is_following value: {type(is_following).__name__}'
+                f'Unexpected follow value: {type(is_following).__name__}'
             )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)